client/src/includes/initStimulus.ts

```diff
@@ -17,13 +17,13 @@ class WagtailApplication extends Application {
    * const content = await controller?.extractContent();
    * ```
    */
-  queryController(identifier: string): Controller | null {
+  queryController<T extends Controller<Element>>(identifier: string) {
```

Comment:
This would only benefit us (because devs won't get the typing automatically). Not sure if this is OK to add during feature freeze, but it's just typing so I think it should be fine.